core/src/filed/dir_cmd.h

```diff
@@ -32,6 +34,9 @@ void* process_director_commands(JobControlRecord* jcr, BareosSocket* dir);
 void* handle_director_connection(BareosSocket* dir);
 bool StartConnectToDirectorThreads();
 bool StopConnectToDirectorThreads(bool wait = false);
+JobControlRecord* NewFiledJcr();
+bool GetWantedCryptoCipher(JobControlRecord* jcr, crypto_cipher_t* cipher);
+void CleanupFileset(JobControlRecord* jcr);
```

Comment:
ditto here and [1]

Reply:
I'll static back the `NewFiledJcr()` but we need the `CleanupFileset()` in `testfind_fd.cc`